0.4/index.bs

```diff
@@ -367,7 +367,7 @@ to the current zarr group. The "path"s MUST be ordered from largest (i.e. highes
 
 Each "datasets" dictionary MUST have the same number of dimensions and MUST NOT have more than 5 dimensions. The number of dimensions and order MUST correspond to number and order of "axes".
 Each dictionary in "datasets" MUST contain the field "coordinateTransformations", which contains a list of transformations that map the data coordinates to the physical coordinates (as specified by "axes") for this resolution level.
-The transformations are defined according to [[#trafo-md]]. The transformation MUST only be of type `translation` or `scale`.
+The transformations are defined according to [[#trafo-md]]. The transformation MUST only be of type `identity`, `translation` or `scale`.
```

Comment:
What if we remove "The transformation MUST only be of type `identity`, `translation` or `scale`." entirely, and just say "The elements of `coordinateTransformations` are the transformations defined in [[#trafo-md]]"? This way, the transformations section of the spec is the single source of truth.

Reply:
:+1: yes added in 1a55fa1.